### PR TITLE
Update docs

### LINKED ISSUE #738: Update docs
**Describe the bug**
Our readmes are outdated at several places and need some love.

**Additional context**
Especially for new devs, keeping the docs about project architecture and entity relations up-to-date reduces feedback loops.

### PR DESCRIPTION
**Pull request by devmount**
## Description of the Change

Fixes markdown formatting issues and updates the project structure and er diagrams.

## Benefits

Synced docs with the current code base

## Applicable Issues

Closes #738

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -75,6 +87,38 @@ erDiagram
...
+  }
+  SUBSCRIBERS ||--o{ INVITES : hold
+  INVITES {
+    int id PK "Unique invite key"
+    int subscriber_id FK "Person who holds this invite"
+    string code "Unique invitation code"
+    enum status "Invitation status [active, revoked]"
+    date time_created "UTC timestamp of invite creation"
+    date time_updated "UTC timestamp of last invite modification"
+    int owner_id FK "Person who creates and owns this invite"
```
Comment (devmount) [inline docs/README.md:112]: @MelissaAutumn It seems the field `owner_id` is duplicate to the field `subscriber_id` - can we remove one of them?

Comment (MelissaAutumn) [reply docs/README.md:112]: I believe owner_id can be an admin. I'd have to recheck the code.

Comment (MelissaAutumn) [reply docs/README.md:112]: Yep. We give users 10 invites they can use, and that's the owner_id (And this is optional for admin use.) Subscriber_id is the created user from an invite.

File: docs/README.md [hunk 2]
```diff
@@ -75,6 +87,38 @@ erDiagram
...
+    enum type "Connection type [zoom, google, fxa, caldav]"
+    string type_id "Universal unique identifier"
+    string token "Passphrase or passtoken for connecting"
+    date time_created "UTC timestamp of connection creation"
+    date time_updated "UTC timestamp of last connection modification"
+  }
+  SUBSCRIBERS ||--o{ INVITES : hold
+  INVITES {
+    int id PK "Unique invite key"
+    int subscriber_id FK "Person who holds this invite"
```
Comment (MelissaAutumn) [inline docs/README.md:107]: User created from this specific invite

File: docs/README.md [hunk 3]
```diff
@@ -75,6 +87,38 @@ erDiagram
...
+    string code "Unique invitation code"
+    enum status "Invitation status [active, revoked]"
```
Comment (MelissaAutumn) [inline docs/README.md:109]: (Note a used invite is still active.)

File: docs/README.md [hunk 4]
```diff
@@ -107,12 +154,16 @@ erDiagram
...
     int earliest_booking "Can't book if it's less than this many minutes before start time"
     int farthest_booking "Can't book if start time is more than this many minutes away"
+    json weekdays "List of selected weekdays (1-7, ISO format)"
     int slot_duration "Size of the Slot that can be booked in minutes"
     date time_created "UTC timestamp of schedule creation"
     date time_updated "UTC timestamp of last schedule modification"
+    string meeting_link_provider "Name of the provider for meeting links (e.g. Zoom)"
+    string slug "Random or customized url part"
+    bool booking_confirmation "True if booking requests need to be confirmed by owner"
+    string timezone "Configured timezone name"
```
Comment (MelissaAutumn) [inline docs/README.md:166]: ...(e.g. America/Vancouver, Europe/Berlin, etc)

File: docs/README.md (hunk 1 above)
Comment (devmount) [reply docs/README.md:112]: Got it. Thanks!

**Pull request merged**